pubsub/drivertest/drivertest.go

```diff
@@ -33,6 +34,11 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+// Options contains settings for RunConformanceTests.
+type Options struct {
+	SkipTestsOfNonExistentThings bool
```

Comment:
Instead of naming this in terms of what it functionally does, it would be clearer to name it in terms of what the underlying system behavior is. Perhaps:

```suggestion
	// DynamicTopics must be set to true if the underlying pub/sub mechanism does
	// not have an explicit "create topic" operation (i.e. publication and subscription occur
	// dynamically).
	DynamicTopics bool
```

Reply:
Done-ish. (DynamicTopicsAndSubscriptions).